src/ar_http_iface_tests.erl

```diff
@@ -4,9 +4,45 @@
 
 -module(ar_http_iface_tests).
 
+-export([can_post_multiple_txs_per_wallet/0, accepts_blocks_with_multiple_txs_per_wallet/0]).
+
 -include("ar.hrl").
 -include_lib("eunit/include/eunit.hrl").
 
+fork_1_7_test_() ->
+	{
+		foreach,
+		fun() ->
+			meck:new(ar_fork),
+			meck:expect(ar_fork, height_1_7, fun() -> 0 end),
+			ok
+		end,
+		fun(ok) ->
+			meck:unload(ar_fork)
+		end,
+		[
+			%% Tests that run for both before and after fork 1.7
+			{"get_last_tx_single_test", fun get_last_tx_single_test/0},
+			{"add_external_tx_test", fun add_external_tx_test/0},
+			{"add_external_tx_with_tags_test", fun add_external_tx_with_tags_test/0},
+			{"find_external_tx_test", fun find_external_tx_test/0},
+			{"fail_external_tx_test", fun fail_external_tx_test/0},
+			{"add_tx_and_get_last_test", fun add_tx_and_get_last_test/0},
+			{"get_subfields_of_tx_test", fun get_subfields_of_tx_test/0},
+			{"get_pending_tx_test", fun get_pending_tx_test/0},
+			{"get_multiple_pending_txs_test_", get_multiple_pending_txs_test_()},
+			{"get_tx_by_tag_test", fun get_tx_by_tag_test/0},
+			{"get_tx_body_test", fun get_tx_body_test/0},
+			{"get_txs_by_send_recv_test_", get_txs_by_send_recv_test_()},
+			%% Tests that only run for after fork 1.7
+			{"can_post_multiple_txs_per_wallet", fun can_post_multiple_txs_per_wallet/0},
```

Comment:
How about replacing the two new tests with a multi-node test doing this:

1. Start both master and slave
2. Push two TXs to master
3. Mine on master
4. Verify new block is on slave and includes both TXs